sdk/netapp/Microsoft.Azure.Management.NetApp/src/Customizations/NetAppResourceOperationsExtensions.cs

```diff
@@ -0,0 +1,331 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Management.NetApp
+{
+    using Microsoft.Rest;
+    using Microsoft.Rest.Azure;
+    using Models;
+    using Newtonsoft.Json;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Extension methods for NetAppResourceOperations.
+    /// </summary>
+    public static partial class NetAppResourceOperationsExtensions
+    {
+        /// <summary>
+        /// Check file path availability
+        /// </summary>
+        /// <remarks>
+        /// Check if a file path is available. Note this operation is deprecated. Please use CheckFilePathAvailability(string location, string name, string subnetId) instead
+        /// </remarks>
+        /// <param name='operations'>
+        /// The operations group for this extension method.
+        /// </param>
+        /// <param name='location'>
+        /// The location
+        /// </param>
+        /// <param name='name'>
+        /// Resource name to verify.
+        /// </param>
+        /// <param name='type'>
+        /// Resource type used for verification. Possible values include:
+        /// 'Microsoft.NetApp/netAppAccounts',
+        /// 'Microsoft.NetApp/netAppAccounts/capacityPools',
+        /// 'Microsoft.NetApp/netAppAccounts/capacityPools/volumes',
+        /// 'Microsoft.NetApp/netAppAccounts/capacityPools/volumes/snapshots'
+        /// </param>
+        /// <param name='resourceGroup'>
+        /// Resource group name.
+        /// </param>
+        //[System.Obsolete("This operation is deprecated. Please use CheckFilePathAvailability(string location, string name, string subnetId) instead.", false)]
```

Comment:
Given the breaking change policy, there should not be a reason to remove this in the future.